zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/config/ApiDescriptionConfig.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.config;
+
+import io.camunda.security.ConditionalOnSaaSConfigured;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
💭 I am starting to wonder if the Swagger UI stuff should go into the `dist` module. At least the creation of the Swagger UI beans and stuff like that. The REST module could still contain business logic to properly build it. We're simply spreading the declaration of Spring beans to more and more components this way and I am not sure if we want that overall. Could be something to discuss more broadly in the orchestration cluster channel.